components/misc_metrics/page_metrics.cc

```diff
@@ -129,6 +142,7 @@ void PageMetrics::ReportAllMetrics() {
   ReportDomainsLoaded();
   ReportPagesLoaded();
   ReportFailedHTTPSUpgrades();
+  ReportBookmarkCount();
   periodic_report_timer_.Start(
       FROM_HERE, base::Time::Now() + kReportInterval,
```

Comment:
counting bookmarks can be expensive (there are users with thousands of them), so let's avoid doing this every 10 seconds

Reply:
my mistake. i changed the intervals for dev purposes and forgot to change them back